.github/workflows/lint_python.yml

```
@@ -0,0 +1,28 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v3
+      - run: sudo apt upgrade -y && sudo apt-get install curl ffmpeg libcurl4-openssl-dev libssl-dev motion ssh v4l-utils -y
+      - run: pip install --upgrade pip wheel
+      - run: pip install bandit black codespell flake8 flake8-2020 flake8-bugbear
+                         flake8-comprehensions isort mypy pytest pyupgrade safety
+      - run: bandit --recursive --skip B104,B105,B108,B110,B301,B310,B321,B324,B402,B403,B404,B602,B603,B605,B607,B701 .
```

```suggestion
      # https://bandit.readthedocs.io/en/latest/plugins/index.html
      - run: bandit --recursive --skip B104,B105,B108,B110,B301,B310,B321,B324,B402,B403,B404,B602,B603,B605,B607,B701 .
```
This makes it easier to lookup the individual test codes and unskip + address them when we find time.